src/app.rs

```diff
@@ -6,7 +6,7 @@ use crate::router::{Endpoint, Router};
 use crate::server::serve;
 use crate::{Error, Middleware};
 
-const DEFAULT_MAX_CONNECTIONS: usize = 256;
+const DEFAULT_MAX_CONNECTIONS: usize = 248;
```

Comment:
Lowering this value seems to stabilize memory usage when the server experiences high traffic. We should consider providing guidance on setting this value, as there is likely a sweet spot that varies based on runtime, operating system, and use case.